addons/mail/models/mail_notification.py

```diff
@@ -9,7 +9,7 @@
 
 class MailNotification(models.Model):
     _name = 'mail.notification'
-    _table = 'mail_message_res_partner_needaction_rel'
+    _table = 'mail_notification'
```

Comment:
Finally!
So do you have to still specify the table name explicitly here now? Since it is now the default table name matching model name.

Reply:
Better explicit than implicit for this one :D